webclient/components/pt-info/single/1time-1row-mField/phq9/view-layer/grid-with-cards-ct.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div>
+    <showContentInCardComponent
+      propMainCardName="phq9"
+      :propChildCardsArray="cfArOfphq9ForDisplay"
+      :propClientSideTableLevelActions="[
+        {
+          actionDescription: 'Multi edit',
+          actionUIByElementIoIconClass: 'el-icon-money',
+        },
+        {
+          actionDescription: 'Toggle card display',
+          actionUIByElementIoIconClass: 'el-icon-remove-outline',
+        },
+        {
+          actionDescription: 'Close card',
+          actionUIByElementIoIconClass: 'el-icon-close',
+        },
+      ]"
+      :propClientSideRowLevelActions="[{}]"
+    />
+  </div>
+</template>
+
+<script>
+import clientSideTblOfPatientPhq9 from '../db/client-side/structure/patient-table-of-phq9.js'
+import showContentInCardComponent from '@/components/pt-info/single/common/show-content-in-card-component.vue'
+
+export default {
+  components: { showContentInCardComponent },
+  computed: {
+    cfArOfphq9ForDisplay() {
+      const arOfObjectsFromClientSideDB = clientSideTblOfPatientPhq9
+        .query()
+        .with('tblPhq9MasterLink')
+        .where('ROW_END', 2147483647.999999)
+        .get()
+      console.log(arOfObjectsFromClientSideDB)
+      for (var i = 0; i < arOfObjectsFromClientSideDB.length; i++) {
+        arOfObjectsFromClientSideDB[i]['cardContentOfTypeStringToShowInBodyOfCards'] =
+          arOfObjectsFromClientSideDB[i].tblPhq9MasterLink.phq9QuestionFullText +
+          ': ' +
+          arOfObjectsFromClientSideDB[i].optionValueChosenByPatient
+      }
+      return arOfObjectsFromClientSideDB
+    },
+  },
+  methods: {
+    mfIconDeleteClickedOnChildCard(pClientSideUniqRowId) {
+      clientSideTblOfPatientPhq9.update({
+        where: pClientSideUniqRowId,
+        data: {
+          ROW_END: Math.floor(Date.now() / 1000),
+        },
+      })
+    },
+    mxOpenMultiEditCtInEditLayer() {
```

Comment:
Type error - function name should be 'mfOpenMultiEditCtInEditLayer' instead of 'mxOpenMultiEditCtInEditLayer'